cmd/kube-proxy/app/server.go

```diff
@@ -357,6 +358,31 @@ func (o *Options) writeConfigFile() error {
 	return nil
 }
 
+func GetBindAddressHostPort(address string, bindPort int) (string, string, string, error) {
```

Comment:
Need a comment for exported function - what is the purpose and what it returns?

Reply:
It will return ``host、port、tag(ipv4 or ipv6)、error``
The param ``bindPort``is used to set a default value or the port you really want to bind when the address does not contain the port.
https://github.com/kubernetes/kubernetes/pull/76320/files#diff-48f6b24dd0fd4d66bfb6caa7523e11ebR220